web/app.js

```diff
@@ -1251,6 +1251,7 @@ var PDFViewerApplication = {
     eventBus.on('rotateccw', webViewerRotateCcw);
     eventBus.on('documentproperties', webViewerDocumentProperties);
     eventBus.on('find', webViewerFind);
+    eventBus.on('findFromUrlHash', webViewerFindFromUrlHash);
```

Comment:
Nit: Please change the event name to lower-case, so that it's consistent with all the existing ones.